src/BTVNanoCommissioning/workflows/ctag_Wctt_valid_sf.py

```diff
@@ -97,20 +99,45 @@ def process(self, events):
     def process_shift(self, events, shift_name):
         dataset = events.metadata["dataset"]
         isRealData = not hasattr(events, "genWeight")
+
+        isMu = False
+        isEle = False
+        if "WcM" in self.selMod or "semittM" in self.selMod:
+            triggers = ["IsoMu27", "IsoMu24"]
+            isMu = True
+            dxySigcut = 0  # 1
+            muNeEmSum = 1  # 0.7
+            muonpTratioCut = 1  # 0.4
```

Comment:
Does the changes on purpose here? 
also muonpTratio  `cut =1` means  no selection?

If I understand correctly, you want to sync selection between e/mu channel?